ng-dev/utils/bazel-bin.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license
+ * Copyright Google LLC
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {join} from 'path';
+import {determineRepoBaseDirFromCwd} from './repo-directory.js';
+
+let bazelPath = process.env.BAZEL;
+const bazel = {
+  get BAZEL_BIN() {
+    if (!bazelPath) {
+      bazelPath = join(determineRepoBaseDirFromCwd(), 'node_modules/.bin/bazel');
```

Comment:
IIRC `@bazel/bazelisk` does expose an export to retrieve the binary? On windows `.bin/bazel` may not exist

Reply:
Unfortunately it doesn't support esmodules currently.

Reply:
Couldn't you import it via interop?